### ISSUE TITLE
Bug in code for l stick config

### ISSUE THREAD

**Issue by timmeh87** [CTCaer/jc_toolkit#26]
its not a huge deal but it was leaving me scratching my head for a while

in jctool.c around line 843, there is an indexing error which results in jctool displaying incorrect factory stick information, the second circled index should be [1]

![image](https://user-images.githubusercontent.com/31858838/47304707-600d7a80-d5f5-11e8-97d0-1e30495960f8.png)

**Comment by CTCaer** [comment]
Good catch. I can't remember these at all, but based on what I wrote here it must be a bug:
https://github.com/CTCaer/Nintendo_Switch_Reverse_Engineering/blob/master/spi_flash_notes.md#analog-stick-factory-and-user-calibration
